.github/workflows/ci.yml

```diff
@@ -27,6 +27,18 @@ jobs:
             - name: Run PHPUnit unit tests
               run: vendor/bin/phpunit
 
+    coding_standards:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@master
+            - uses: shivammathur/setup-php@master
+              with:
+                  php-version: '8.0'
+            - name: Install dependencies
+              run: composer update
```

Comment:
Shouldn't this be composer install?

Reply:
This is a library that doesn't use lock file. In this case it's advised by composer to always run `update`. (try running `composer install` without a lock file, it will tell you)